pkg/clierrors/clierrors.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package clierrors
+
+import "errors"
+
+var (
+	ErrNoBlockchainID                 = errors.New("failed to find the blockchain ID for this subnet, has it been deployed/created on this network?\nyou can use 'avalanche blockchain import' if having partial information on a deployed subnet")
```

Comment:
no blockchain ID found. Use 'avalanche blockchain import' to provide the required Subnet details

Reply:
blckchain ID can also be absent for the blockchain not being deployed. np changing if you also include reference to this option

Reply:
Let's get Javier's opinion on this text (you'll have to ping him on slack)

Reply:
No `blockchainID` found. To resolve this:
- Use `avalanche blockchain deploy` to deploy the blockchain and generate a `blockchainID`.
- Or use `avalanche blockchain import` to import an existing configuration.

Reply:
will use latest one